app/main.py

```diff
@@ -1057,22 +1086,44 @@ def display_alert_modal(n_clicks):
 
 
 @app.callback(
-    Output({"type": "alert_frame", "index": MATCH}, "src"),
-    Input({"type": "alert_slider", "index": MATCH}, "value"),
-    State({"type": "individual_alert_frame_storage", "index": MATCH}, "children"),
+    [
+        Output({"type": "alert_frame", "index": MATCH}, "src"),
+        Output({"type": "alert_bbox_container", "index": MATCH}, "children"),
+    ],
+    [
+        Input({"type": "alert_slider", "index": MATCH}, "value"),
+        Input({"type": "bbox_toggle_checklist", "index": MATCH}, "value"),
+    ],
+    [
+        State({"type": "individual_alert_frame_storage", "index": MATCH}, "children"),
+        State("images_boxes_live_alerts", "data"),
+    ],
 )
-def select_alert_frame_to_display(slider_value, urls):
-    """
-    --- Choosing the alert frame to be displayed in an alert modal ---
-
-    When the user has opened an alert modal, he or she can choose the alert frame to view thanks to the slider. This
-    callback is triggered by a change in value of the slider and return the URL address of the frame to be displayed.
-    Like there is one alert modal per event, there is one alert slider per event, which allows to use MATCH here.
-    """
-    if slider_value is None:
+def select_alert_frame_to_display(slider_value, bbox_toggle_value, urls, bboxes_dict):
+    event_id = dash.callback_context.inputs_list[0]["id"]["index"]
+    # Check if the urls for the specific event_id exists
+    if slider_value is None or urls is None or event_id not in urls:
         raise PreventUpdate
```

Comment:
Since this last commit, after running some test it seems that that this addon raises PreventUpdate and then frame remains the same + there is no bbox drawn.